recipe/meta.yaml

```diff
@@ -64,8 +64,8 @@ outputs:
         - libcurand-dev      # [(cuda_compiler_version or "").startswith("12")]
       run:
         - python
-        - {{ pin_compatible('numpy') }}
-        - {{ pin_compatible('scipy') }}
+        - numpy
+        - scipy >=0.16
```

Comment:
Would think these would be needed given the usage of SciPy's Cython BLAS API above

Reply:
Do you mean that we need the pin_compatible here because of binary compatibility between the build and runtime environment?

I'm *hoping* that the Scipy Cython BLAS ABI is stable between versions - and that we should be able to build against a newer scipy version and then run with an older one for instance - but I haven't tried it out

Reply:
Sorry for dropping the ball here

Yeah would think the SciPy Cython BLAS would require the `pin_compatible` strategy given the potential risk of ABI breakage (though also have not tested this assumption thoroughly so could be wrong about that). We could pick an old-ish version of SciPy to build against and that would keep the range of versions pretty flexible (if that is the goal)

Looking at NumPy more closely, agree we don't need `pin_compatible` for NumPy. It doesn't appear we link to it. Also as `numpy` is not explicitly in `host`, we might end up getting the latest version of `numpy` during the build and then have an overly restrictive recent `numpy` requirement in `run`, which would cause install issues. So think we should drop that. Picking that change up here: https://github.com/conda-forge/implicit-feedstock/pull/81/commits/cb082fd7b4accf7b6f70be489db87b0cf86a2619